### ISSUE TITLE
Query parameters

### ISSUE THREAD

**Issue by en0mia** [en0mia/stuquiz-api#25]
The `execute` function expects the query parameters to be tuples, while sometimes they're passed as strings.

For instance this:

```
    def delete_answer(self, answer: Answer) -> bool:
        query = "DELETE FROM answer WHERE id = %s"
        return self.delete(query, answer.id)
```

Should be:
```
    def delete_answer(self, answer: Answer) -> bool:
        query = "DELETE FROM answer WHERE id = %s"
        return self.delete(query, (answer.id, )) # <-------
```

We should fix it ASAP because at this point most of the repositories are not working properly.

:rotating_light: **Always remember to test the APIs before opening the PR (i.e. by means of tools like Postman, Insomnia, etc etc)**

**Comment by lorenzovarese** [comment]
Thanks for pointing it out. I took charge of it.

**Comment by en0mia** [comment]
Closed by PR #30
